src/lib/tls/tls_policy.cpp

```diff
@@ -585,6 +593,17 @@ void print_vec(std::ostream& o, const char* key, const std::vector<Group_Params>
    o << "\n";
 }
 
+void print_vec(std::ostream& o, const char* key, const std::vector<Certificate_Type>& types) {
```

Comment:
Use `std::string_view` as type for key?

Reply:
Yeah, we could certainly do that. Though, there are more implementations of `print_vec` here, that I wanted to stay consistent with.